OpenSim/Common/Reporter.h

```diff
@@ -163,10 +163,24 @@ OpenSim_DECLARE_CONCRETE_OBJECT_T(TableReporter_, InputT, Reporter<InputT>);
     TableReporter_() = default;
     virtual ~TableReporter_() = default;
 
+    /** Retrieve the report.                                                  */
```

Comment:
I think this user-facing documentation could be improved by providing an example that helps the user understand what `_<ValueT>` means. There is already an issue to track the lack of meaningful documentation (#1068), so we may just want to leave this for later.